### DISCUSSION TITLE
Custom 1.8.1 build needed

### DISCUSSION THREAD

**Discussion by ghost** [Sporif/dxvk-async#10] [Q&A] [answered]
Hi guys. Is someone able to build 1.8.1 Async (up to the most recent commit in master)including the improvements made from doitsujin/dxvk#1582 

Thank you for your help 🙂

@rbernon @Sporif @Kron4ek

**Comment by Kron4ek** [comment]
Latest master (2f553b5) + async + [PR #1582](https://github.com/doitsujin/dxvk/pull/1582).

[dxvk-async-1.8.1-2f553b5.tar.gz](https://github.com/Sporif/dxvk-async/files/6136278/dxvk-async-1.8.1-2f553b5.tar.gz)

**Reply by ghost** [reply]
@Kron4ek hi thanks so much for your help with this. I was after the most recent dxvk master patched with Async due to an issue fixed with the most recent commit with some of the dx9 games I play. Are you able to kindly post the the latest commit patched ? Thank you 🍀🍀🍀

**Reply by Kron4ek** [reply]
[dxvk-async-1.8.1-b84a03b.tar.gz](https://github.com/Sporif/dxvk-async/files/6453941/dxvk-async-1.8.1-b84a03b.tar.gz)

**Reply by ghost** [reply]
Thank you very much 🍀🍀🙏🙏😁

**Comment by Ryusennin** [comment]
Hi Kron4ek,

Could you please release a new build with the latest commits for Days Gone support?

Thanks a lot for your great patch.

**Reply by Kron4ek** [reply]
[dxvk-async-1.8.1-c510800.tar.gz](https://github.com/Sporif/dxvk-async/files/6523349/dxvk-async-1.8.1-c510800.tar.gz)

**Comment by ghost** [comment, marked as answer]
@Kron4ek 

Hi again. I was wondering if I could get the most recent dxvk commit patched with Async. I'm running some tests with frame latency on Windows with Vulkan.

Thank you and have a great day 🙏

**Reply by Kron4ek** [reply]
[dxvk-async-1.8.1-01033af.tar.gz](https://github.com/Sporif/dxvk-async/files/6626679/dxvk-async-1.8.1-01033af.tar.gz)
